### ISSUE TITLE
Add drag handle icon to playlist view

### ISSUE THREAD

**Issue by Chocobozzz** [Chocobozzz/PeerTube#5243]
### Describe the problem to be solved

People may not be aware that you can reorder the playlist in the playlist page (`/my-library/video-playlists/...`)

### Describe the solution you would like

Add a drag handle icon like on youtube 
![screen_2022-09-07-16:11:18](https://user-images.githubusercontent.com/5180488/188900062-71bcb887-03aa-4550-8b05-aa86192f288c.png)

**Comment by xundeenergie** [comment]
especially on smartphone-view!

**Comment by okaziya** [comment]
@Chocobozzz Being able to reorder/rank playlists would be a great feature! We're struggling with this in OwnTube.tv client apps for PeerTube. Would the re-ordering mentioned here also impact how the playlists are returned via the Playlists API?

**Comment by Chocobozzz** [comment]
> Being able to reorder/rank playlists would be a great feature! We're struggling with this in OwnTube.tv client apps for PeerTube. Would the re-ordering mentioned here also impact how the playlists are returned via the Playlists API?

No, this feature is just an UX addition. I created another issue for your feature request: https://github.com/Chocobozzz/PeerTube/issues/6928
